docs/01-Using-Fleet/standard-query-library/standard-query-library.yml

```diff
@@ -992,4 +992,14 @@ spec:
   tags: compliance, CIS, CIS18.9.108.2.1
   platform: windows
   contributors: defensivedepth
-
+---
+apiVersion: v1
+kind: policy
+spec:
+  name: Identify Apple development secrets (macOS)
+  query: SELECT * FROM keychain_items WHERE label LIKE '%ABCDEFG%';
+  description: "Identifies certificates associated with Apple development signing and notarization. Replace ABCDEFG with your company's identifier."
+  resolution: "Ensure your official Apple builds, signing and notarization happen on a centralized system, and remove these certificates from workstations."
```

Comment:
What do you mean by "centralized system"? Any examples or docs we can link to here?

Reply:
https://security.stackexchange.com/questions/106867/centralized-key-management-for-ios-and-android-code-signing

First answer here seems to say that it's ok for the developer key to be used locally? You're the expert, but I just wanna double check on that here. If it is ok, then will this query unnecessarily catch those?

Reply:
Sounds good to me.